_rules/SC1-2-video-media-alternative.md

```diff
@@ -0,0 +1,128 @@
+---
+name: Video as a media alternative for text
+description: |
+  This rule checks non-streaming `video` is a media alternative for text on the page.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output](#audio-output)
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element that is [visible](#visible), where the video  contains audio.
+
+### Expectation 1
+
+All the information contained in each target element is available as text (directly or via text alternatives) that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
does "text alternatives" need to be specified (or defined)?

Reply:
Now that I think about it mentioning text alternatives doesn't really add anything. We can simply remove "(directly or via text alternatives)" and it would work just fine I think. Do you agree?